### DISCUSSION TITLE
Elevate notebook reset issue

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#771] [Platform]
Hey all, 

I wanted to elevate[ this issue](https://github.com/MAAP-Project/Community/issues/735) as a priority.  I've been getting a reset of my notebook every 7 mins or so, which is making coding frustrating. Is there any way to make this happen less frequently? This issue is why I stopped working with the new ADE. Now that the new ADE is the only option are there ways to work around this? 

Thanks!

**Comment by wildintellect** [comment]
@mccabete Which workspace(s) does this occur in? As @rtapella noted in the ticket, creating a fresh workspace that doesn't have the collaboration tool might solve this issue.

**Reply by mccabete** [reply]
Hey @wildintellect -- I responded in the ticket.

**Comment by rtapella** [comment]
I thought we had already turned off the collaboration tool, but it seems that fix is not deployed to ops yet.

**Reply by mccabete** [reply]
Wait @rtapella, does "ops" refer to `https://ade.maap-project.org/` or `https://ade.ops.maap-project.org/`? Because `https://ade.ops.maap-project.org/` does not give me any issues, but `https://ade.maap-project.org/` does. Your comment suggests that the issue shouldn't be happening on `https://ade.maap-project.org/`  and should be happening on  `https://ade.ops.maap-project.org/`.

**Reply by wildintellect** [reply]
@mccabete not quite, they are NEW Ops and OLD Ops (ade.ops). This issue should only be in NEW Ops, but should not exist if you start a new workspace. Older workspaces (a few weeks ago) had a buggy feature enabled that should be turned off soon...

**Reply by rtapella** [reply]
Yes. Even newer workspaces on new-ops (ADE.maap-project.org) will have the bug. The dev team is currently discussing a small release this week to fix some of the critical issues like the jumping-cursor and a few other things. We will let you know when we have timing/plan fully in place.

**Reply by mccabete** [reply]
@rtapella I can confirm from my end that even a newly-generated workspace in the new ADE has this cursor-moving kernel-restarting bug. This is in line with your last comment, so no new intel.

**Reply by rtapella** [reply]
Thank you for testing it!

**Reply by wildintellect** [reply]
Thanks @mccabete I've put it on the agenda for a meeting this afternoon.

**Reply by wildintellect** [reply]
New release is slated for this week, notice will be posted in announcements, and email sent to all users.

**Comment by bsatoriu** [comment]
This reset issue has been resolved in [version 3.1.0](https://github.com/orgs/MAAP-Project/discussions/790) released today,
